modules/analyzers/git-analyzer/main.go

```diff
@@ -26,6 +28,11 @@ type Revision struct {
 }
 
 type GitAnalyzer struct {
+	workdirs []string
+	pkgInfo  packageInfo
+}
+
+type packageInfo struct {
```

Comment:
Why do we need a new struct here? Also the struct's name is totally misleading. This has nothing to do with a package when it holds only revision stuff.